recipe/meta.yaml

```
@@ -8,28 +8,32 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: http://www.openssl.org/source/{{ name }}-{{ version }}.tar.gz
-  sha256: 50a98e07b1a89eb8f6a99477f262df71c6fa7bef77df4dc83025a2845c827d00
+  sha256: fc20130f8b7cbd2fb918b2f14e2f429e109c31ddd0fb38fc5d71d9ffed3f9f41
 
 build:
-  number: 1002
-  no_link: lib/libcrypto.so.1.0.0        # [linux]
-  no_link: lib/libcrypto.1.0.0.dylib     # [osx]
+  number: 0
```

@hmaarrfk should we bump back to 1000, rather than 0?

```suggestion
  number: 1000
```